File: lib_util/src/main/java/com/pds/util/StringUtils.java
package com.pds.util;

/**
 * @author: pengdaosong.
 * CreateTime:  2018/12/9 3:43 PM
 * Email：pengdaosong@medlinker.com.
 * Description:
 */
public class StringUtils {

    public static boolean isStrictEmpty(String s) {
        return s == null || "".equals(s) || "null".equalsIgnoreCase(s);
    }
}

File: study/kotlin_demo/src/main/java/com/pds/demo/expand/StringUtils.kt
package com.pds.demo.expand

import android.text.TextUtils
import java.util.regex.Pattern

/**
 * @author: pengdaosong
 * @CreateTime:  2020/12/21 9:27 AM
 * @Email: pengdaosong@medlinker.com
 * @Description:
 */
fun String.isPhoneNo() : Boolean{
    if (TextUtils.isEmpty(this)) {
        return false
    }
    val p = Pattern.compile("^((1[0-9][0-9])\\d{8}$)")
    val m = p.matcher(this)
    return m.matches()
}
